README.md

```diff
@@ -1 +1,43 @@
-# node-arguments-lab
\ No newline at end of file
+# Node Arguments Lab
+
+In this lab, you will pass and access command-line arguments when working with Node.
+
+## Objectives
+
+1. Implement a Node script that takes CLI arguments and prints them back
+1. Use two arguments
+
+## Introduction
+
+Knowing how to pass and access command-line arguments is important in building CLI tools or small programs like the program which gives a random password.
+
+![](password.png)
+
+## Instructions
+
+1. Implement a Node script `password.js` which takes CLI arguments and prints random passwords based on the arguments
+1. Use two arguments: length (first) and a number of passwords (second) to return so that `node password 2 2` and `node password 10 3` are valid commands
```

Comment:
what do you mean number of passwords? `node 2 2` will give me two passwords, each with two letters?